src/server/data/database.js

```diff
@@ -163,17 +163,52 @@ export function removeBoard(id) {
 }
 
 export function getCardLists(boardId) {
-  return cardLists.filter((list) => list.boardId == boardId);
+  return cardLists.filter((list) => list.boardId === boardId);
 }
 
 export function getCardList(id) {
-  return cardLists.filter((list) => list.id == id)[0];
+  return cardLists.filter((list) => list.id === id)[0];
 }
 
+// Ordered by cardListRank
 export function getCards(cardListId) {
-  return cards.filter(card => card.cardListId === cardListId);
+  let c = cards.filter(card => card.cardListId === cardListId);
+  c = c.sort((a, b) => {
```

Comment:
I think it's important to sort on the client until we are offering reliable sorting on the server side via query params. If we don't have it on the client and decide to change it on the server (when we do in fact start to offer query params for that sort of thing) the client may do weird things.